frame/nfts/src/lib.rs

```diff
@@ -988,18 +1023,60 @@ pub mod pallet {
 				let permitted = check == collection_details.admin || check == details.owner;
```

Comment:
Let's allow anyone to remove the expired approval like I did here: https://github.com/paritytech/substrate/commit/a705c225223ccf3d3d1715ecaf21b054c33a6a10#diff-b7ad00e29ca4365847b516304372848ad6d8770885da79256a1408968155c76a